src/main/java/org/broadinstitute/hellbender/utils/test/VariantContextTestUtils.java

```diff
@@ -18,6 +26,29 @@
 
     private VariantContextTestUtils() {}
 
+    /**
+     * Returns a list of VariantContext records from a VCF file
+     *
+     * @param vcfFile VCF file
+     * @return list of VariantContext records
+     * @throws IOException if the file does not exist or can not be opened
+     */
+    public static List<VariantContext> getVariantContexts(final File vcfFile) throws IOException {
+        final VCFCodec codec = new VCFCodec();
+        try (final InputStream in = new PositionalBufferedStream(( new FileInputStream(vcfFile)))) {
```

Comment:
Yeah, I'm not sure why I was doing this... This method came from somewhere.... Maybe it lives in protected somewhere?